Crash Report 3.0.3: FileUtils.getFileCacheDir NPE

**Describe the bug**
```
Caused by: kotlin.KotlinNullPointerException: 

 
  at org.kiwix.kiwixmobile.utils.files.FileUtils.getFileCacheDir (FileUtils.java:44)

 
  at org.kiwix.kiwixmobile.utils.files.FileUtils.deleteCachedFiles (FileUtils.java:50)

 
  at org.kiwix.kiwixmobile.main.MainActivity.onDestroy (MainActivity.java:775)
```

**Expected behavior**
No crash

**Environment**
- Version of Kiwix Android : 3.0.3
- Device : Huawei Y7 Prime 2019 (HWDUB-Q)
- OS version : Android 8.1

--- a/app/src/main/java/org/kiwix/kiwixmobile/utils/files/FileUtils.kt
+++ b/app/src/main/java/org/kiwix/kiwixmobile/utils/files/FileUtils.kt
@@ -41,15 +41,15 @@ object FileUtils {
     "${Environment.getExternalStorageDirectory()}${File.separator}Android" +
       "${File.separator}obb${File.separator}${BuildConfig.APPLICATION_ID}"
 
-  @JvmStatic fun getFileCacheDir(context: Context): File =
+  @JvmStatic fun getFileCacheDir(context: Context): File? =
     if (Environment.MEDIA_MOUNTED == Environment.getExternalStorageState()) {
-      context.externalCacheDir!!
+      context.externalCacheDir
     } else {
       context.cacheDir
     }
 
   @JvmStatic @Synchronized fun deleteCachedFiles(context: Context) {
-    getFileCacheDir(context).deleteRecursively()
+    getFileCacheDir(context)?.deleteRecursively()
   }
 
   @JvmStatic @Synchronized fun deleteZimFile(path: String) {

